Draft oneway

wtedeschi opened the pull request:
Hi Kurt,

Here we have two bug fixes and 2 new functionalities:
1. Log is enabled also on the mirror side
2. Oneway latency is included with the logging.

For one way i have tested also the AO and AE security, both worked with the current code.

Thanks,
/Fred

shifty91 commented on src/rtc_thread.c:
```diff
@@ -104,6 +105,7 @@ static int rtc_gen_and_send_frames(struct thread_context *thread_context, int so
 		frame_config.frame_length = app_config.rtc_frame_length;
 		frame_config.num_frames_per_cycle = app_config.rtc_num_frames_per_cycle;
 		frame_config.sequence_counter = sequence_counter_begin + i;
+		frame_config.tx_timestamp = ts_to_ns(&tx_time);
```
This `tx_time` is always zero. It misses the call to `clock_gettime()`

shifty91 commented on src/stat.c:
```diff
@@ -200,50 +228,64 @@ static void stat_frame_received_per_period(enum stat_frame_type frame_type, uint
+	stat_update_min_max(oneway_time, &stat->oneway_min, &stat->oneway_max);
+
+	if (stat_frame_type_is_real_time(frame_type) && oneway_time > rtt_expected_rt_limit / 2)
+		stat->oneway_outliers++;
```
Nice, outliers are detected. However, they're not logged.

shifty91 reviewed (changes requested):
Hi Fred,

nice work. Can you address the following overall issues? Then, I'll merge it.

- Adjust the Tx time stamp on `mirror` side (to make one way measurements symmetrical)
- Run `clang-format`
- Fix build errors and warnings
- Stick to RCT

wtedeschi commented:
Hi Kurt,

Thanks for the quick review!
I have addressed all the comments, I will be testing by Monday and submitting an update.

Best regards,
/Fred

shifty91 commented:
@wtedeschi Applied, thank you.
